.github/workflows/playwright.yml

```
@@ -0,0 +1,36 @@
+name: E2E Tests
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
+
+jobs:
+  test-playwright:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
```

Recommended usage at https://github.com/actions/setup-node:

```suggestion
          node-version: 18
```